src/citrine/resources/data_concepts.py

```diff
@@ -702,28 +728,32 @@ def list_by_tag(self, tag: str, per_page: int = 100) -> Iterator[ResourceType]:
             params=params)
         return (self.build(raw) for raw in raw_objects)
 
-    def delete(self, uid: Union[UUID, str], scope: str = CITRINE_SCOPE, dry_run: bool = False):
+    def delete(self, uid: Union[UUID, str, LinkByUID, BaseEntity],
+               scope: Optional[str] = None, dry_run: bool = False):
         """
-        Delete the element of the collection with ID equal to uid.
+        Delete an element of the collection by its id.
 
         Parameters
         ----------
-        uid: Union[UUID, str]
-            The ID.
-        scope: str
+        uid: Union[UUID, str, LinkByUID, BaseEntity]
+            A representation of the object (Citrine id, LinkByUID, or the object itself)
+        scope: Optional[str]
+            [DEPRECATED] use a LinkByUID to specify a custom scope
             The scope of the uid, defaults to Citrine scope (CITRINE_SCOPE)
         dry_run: bool
             Whether to actually delete the item or run a dry run of the delete operation.
             Dry run is intended to be used for validation. Default: false
 
         """
-        path = self._get_path() + "/{}/{}".format(scope, uid)
+        link = _make_link_by_uid(uid)
+        path = self._get_path() + "/{}/{}".format(link.scope, link.id)
```

Comment:
This code is subject to issues if the scope or id contain [URI Reserved Characters](https://en.wikipedia.org/wiki/Percent-encoding).  I don't know if the backend is doing it properly, but we should be sending percent-encoded strings down the wire.

Reply:
That issue seems like it's already present, and made neither worse nor better by this PR

Reply:
I agree, but this is arguably something that should be addressed before a v1 release.  Customers have hit it when they incorporated `/` into their scopes.  Not blocking.